packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -655,6 +658,56 @@ class DropdownButton<T> extends StatefulWidget {
   /// {@endtemplate}
   final ValueChanged<T> onChanged;
 
+  /// A builder to generate custom dropdown buttons corresponding to the
```

Comment:
You might reconsider the terminology here. The dropdown button includes the down-arrow icon. So we're not generating the button, although we're certainly customizing it.